src/components/BackButton/backbutton.css

```diff
@@ -0,0 +1,5 @@
+.backButton {
+    color: rgba(36, 36, 36, 0.6);
+    font-size: 20px;
+    width: 60px;
```

Comment:
don't specify the width for the back button div 
and trying to add 
```css
 cursor: pointer;
```

Reply:
Done